### PR TITLE
Add custom-carbon-log-appender project.

### PR DESCRIPTION
**Pull request by vihanga-liyanage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+```
+# <project title>.
+## Introduction / Use case
+Explained the basic use case of the project. Include examples, external links, public jiras and other helpful material is possible.
+
+## Applicable product version(s)
+Mentioned the list of applicable IS versions.
+
+## How to use
+Provide a step by step guide on using this sample. Include steps on building the project, applying to Identity Server, neccessary configurations, etc.
```
Comment (isharak) [inline README.md]: Better to break this in to two.
How to configure
How to test 
------- Sample request and response / Or steps

File: custom-carbon-log-appender/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Originlly desinged for WSO2 Identity Server version 5.1.0
+
+## How to use
+1. Build the project.
+2. Copy the JAR file in ```target``` directory into ```<IS_HOME>/repository/components/dropins``` directory.
+3. Open the file *<IS_HOME>/repository/conf/log4j.properties* file in a text editor and change,
+  ```log4j.appender.CARBON_LOGFILE=org.wso2.carbon.utils.logging.appenders.CarbonDailyRollingFileAppender```
+  to
+  ```log4j.appender.CARBON_LOGFILE=org.wso2.carbon.custom.utils.logging.appenders.CarbonDailyRollingSanitizedFileAppender```
+4. Restart the server.
```
Comment (isharak) [inline custom-carbon-log-appender/README.md]: What are the action we should do on Identity server to print those logs?

File: custom-carbon-log-appender/pom.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon</groupId>
+    <artifactId>org.wso2.carbon.custom.utils</artifactId>
```
Comment (isharak) [inline custom-carbon-log-appender/pom.xml]: Better to use different artifact ID to represent usecase 
Eg: logappender

File: custom-carbon-log-appender/pom.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (isharak) [inline custom-carbon-log-appender/pom.xml]: Add a new line

File: custom-carbon-log-appender/pom.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Vendor>WSO2 Inc</Bundle-Vendor>
+                        <Bundle-SymbolicName>org.wso2.carbon.custom.utils</Bundle-SymbolicName>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (isharak) [inline custom-carbon-log-appender/pom.xml]: Better if you can avoid dynamic imports

File: custom-carbon-log-appender/target/classes/META-INF/MANIFEST.MF [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
```
Comment (isharak) [inline custom-carbon-log-appender/target/classes/META-INF/MANIFEST.MF]: Better to remove target folder

File: custom-carbon-log-appender/README.md (hunk 2 above)
Comment (vihanga-liyanage) [reply custom-carbon-log-appender/README.md]: I added a patch with logs. Should we add that jar to the repo as well?

**Pull request merged**